src/index.js

```diff
@@ -1,14 +1,21 @@
 // @flow
 import SwaggerUIStandalonePreset from "swagger-ui";
-import "swagger-ui/dist/swagger-ui.css";
+import "../node_modules/cookieconsent/build/cookieconsent.min.js";
```

Comment:
can you do `cookieconsent/build/cookieconsent.min.js`?

Reply:
or wherever is their entry point

Reply:
done